### DISCUSSION TITLE
Fixing "Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')"

### DISCUSSION THREAD

**Discussion by swichers** [IamWarHead/ddb-game-log#108] [General]
This is reported with Foundry v10 and v2.0.0 of the plugin.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')
[Detected 1 package: ddb-game-log]
    at _0x2888e2.getActorByDDBEntityId (main.js:9:1044848)
    at async _0x2888e2.handlePendingRoll (main.js:9:1045884)
    at async _0x2888e2.onClientRoll (main.js:9:1084866)
```

The fix (for me) was to go into the module config and manually set the mappings. This seems to be a new feature and doesn't look like the data migrated properly. Posting as a FYI for anyone else who might arrive here.

**Comment by IamWarHead** [comment]
Hey there @swichers, could you please get in touch with me on discord please?
